### PR TITLE
location fix + Stefan's style changes

### PR DESCRIPTION
**Pull request by profjsb**
Fixes a typo in `HeaderContent.jsx.template` which caused the strange formatting and adds @stefanv's style changes from #732.

### CODE REVIEW

**Comment by stefanv** [comment]
This doesn't do the moving over to the sidebar, does it? My styling changes were meant for that move.

I guess the tests on that one may have failed because the component changed places?

File: static/js/components/HeaderContent.jsx.template [hunk 1]
```diff
@@ -5,19 +5,17 @@ import Box from "@material-ui/core/Box";
...
 import ProfileDropdown from "./ProfileDropdown";
 import Logo from "./Logo";
-import QuickSearchBar from "./QuickSearchBar";
 
 import styles from "./Main.css";
 
 const HeaderContent = () => (
   <div className={styles.topBannerContent}>
-    <div style= {%- raw -%} {{display: "inline-flx", flexDirection: "row" }} {%- endraw -%} >
+    <div style= {%- raw -%} {{display: "inline-flex", flexDirection: "row" }} {%- endraw -%} >
```
Comment (stefanv) [inline static/js/components/HeaderContent.jsx.template:13]: No longer necessary?

Comment (profjsb) [reply static/js/components/HeaderContent.jsx.template:13]: It is -- need to spell `flex` whrite!

**Comment by profjsb** [comment]
> I think the following fix is quite an important one too:
> 
> https://github.com/skyportal/skyportal/pull/732/files#diff-8e6371e901a30fdd0767925dd55ba088R64
> 
> (This prevents the quicksearch from messing with filtering results.)
> 
This isn't defined in `sources.js` -- do we need to create a path for that in `sources.js`?

File: static/js/components/HeaderContent.jsx.template (hunk 1 above)
Comment (stefanv) [reply static/js/components/HeaderContent.jsx.template:13]: I meant, is that whole line still needed :)

**Comment by stefanv** [comment]
> > I think the following fix is quite an important one too:
> > https://github.com/skyportal/skyportal/pull/732/files#diff-8e6371e901a30fdd0767925dd55ba088R64
> > (This prevents the quicksearch from messing with filtering results.)
> 
> This (`FETCH_AUTOCOMPLETE_SOURCES`) isn't defined in `sources.js` -- do we need to create a path for that in `sources.js`?

Yes, problem is that FETCH_SOURCES is handled, and causes the interaction with the filter I described.  By sending an unknown event, nothing happens on the frontend (correct behavior).

**Pull request merged**